### ISSUE TITLE
urllib2.URLError: <urlopen error [Errno 54] Connection reset by peer>

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-bot-old#96]
I get this error about 50% of the time.  I then have to restart the bot. It's perhaps related to #95, which is the same behavior.

``` pytb
/Users/aaronmeurer/.sympy/sympy-bot.conf
> Using /Users/aaronmeurer/.sympy/sympy-bot.conf
> Checking username and password ...
>     OK.
> Working directory: /var/folders/wc/dppcpmxs1tlb36nqcw853wkm0000gn/T/sympy-bot-tmpUikPsd
Traceback (most recent call last):
  File "./sympy-bot", line 337, in <module>
    main()
  File "./sympy-bot", line 130, in main
    review(n, options, username, password)
  File "./sympy-bot", line 258, in review
    pull = github_get_pull_request(config.repository, n)
  File "/Users/aaronmeurer/Documents/Python/sympy/sympy-bot/utils.py", line 121, in github_get_pull_request
    data = json.load(urlopen(url % n))
  File "/sw/lib/python2.7/urllib2.py", line 126, in urlopen
    return _opener.open(url, data, timeout)
  File "/sw/lib/python2.7/urllib2.py", line 394, in open
    response = self._open(req, data)
  File "/sw/lib/python2.7/urllib2.py", line 412, in _open
    '_open', req)
  File "/sw/lib/python2.7/urllib2.py", line 372, in _call_chain
    result = func(*args)
  File "/sw/lib/python2.7/urllib2.py", line 1199, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/sw/lib/python2.7/urllib2.py", line 1174, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 54] Connection reset by peer>
```

**Comment by asmeurer** [comment]
I think this might have something to do with running two instances of SymPy bot one right after another, so that one resets the other.

**Comment by asmeurer** [comment]
Hmm, I thought typing "Closes #96" in the commit message was supposed to automatically close the issue.  Maybe I got the syntax wrong.
